mobile/library/java/org/chromium/net/impl/CronvoyLogger.java

```diff
@@ -0,0 +1,42 @@
+package org.chromium.net.impl;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import io.envoyproxy.envoymobile.engine.types.EnvoyLogger;
+
+final class CronvoyLogger implements EnvoyLogger {
```

Comment:
Question: did you intend to include this class as part of this PR?

Reply:
haha nope, just realized that and removed